docker-compose.yaml

```diff
@@ -1,19 +1,52 @@
 # This is an example compose file that can be used for local development.
 # Modify the config file in `volumes` to point to the desired environment, default id dev.
-#
+# Ensure that the following configuration files have been created in the correct locations:
+#   * public/config.json
+#   * ca-bundle.crt
+#   * server.crt
+#   * server.key
+#   * site.conf
+# Ensure that /etc/hosts contains an entry for:
+# 	127.0.0.1	local.broadinstitute.org
 # Build:
 #   docker build . -t duos
 # Run:
-#   docker-compose up
+#   docker compose up -d
 # Test
-#   http://localhost:3000/
+#   https://local.broadinstitute.org/
 version: '3.8'
 services:
   app:
     image: duos:latest
     container_name: duos
     ports:
-      - 3000:8080
+      - 8080:8080
     volumes:
-      - ./config/perf.json:/usr/share/nginx/html/config.json
+      - ./public/config.json:/usr/share/nginx/html/config.json
     command: ["nginx", "-g", "daemon off;"]
+    restart: always
+
+  proxy:
+    image: broadinstitute/openidc-proxy:tcell_3_1_0
+    container_name: proxy
```

Comment:
Rename this to something else (like `duos-proxy`). If someone has a local container for consent then Docker will throw an error stating that the name `proxy` is already in use.